CONTRIBUTING.md

```diff
@@ -41,17 +41,17 @@ TensorFlow coding style.
 #### General guidelines and philosophy for contribution
 
 * Include unit tests when you contribute new features, as they help to
-  a) prove that your code works correctly, b) guard against future breaking
+  a) prove that your code works correctly, and b) guard against future breaking
   changes to lower the maintenance cost.
 * Bug fixes also generally require unit tests, because the presence of bugs
-  usually indicates insufficient test coverage.
+  usually indicate insufficient test coverage.
```

Comment:
This was correct before since the subject is "presence".

Reply:
You're right. Sorry about that.